yoti-sdk-api/src/main/java/com/yoti/api/client/DateTime.java

```diff
@@ -32,6 +36,17 @@ public static DateTime from(long microseconds) {
         return new DateTime(Date.from(calendar), Time.from(calendar, mod));
     }
 
+    public static DateTime from(String dateTimeStringValue) throws DateTimeParseException {
```

Comment:
I'm not sure this public method should be returning null if gets an invalid string (i.e. null or empty).

What's the background to this change?  What's the context?